---
permalink: /cv/
title: "CV"
excerpt: "CV"
author_profile: true
redirect_from: 
  - /cv
  - /cv.html
---

Dr Robert S Young  
Centre for Global Health, Usher Institute, Usher Building, University of Edinburgh, 5-7 Little France Road, Edinburgh BioQuarter – Gate 3, Edinburgh, EH16 4UX, UK  
Zhejiang University - University of Edinburgh (ZJE) Institute, Zhejiang University - International Campus, 718 East Haizhou Road, Haining, P.R. China  

EMPLOYMENT

May 2019 – date,	Biomedical Sciences Teaching Organisation	
•	Lecturer (Zhejiang)	University of Edinburgh	

Jan 2012 – April 2019, MRC Human Genetics Unit, University of Edinburgh
•	Post-doctoral Research Associate
•	Supervisor: Prof Martin Taylor

Aug 2011 – Dec 2011, MRC Functional Genomics Unit, University of Oxford
•	Pre-doctoral Research Associate	MRC 
•	Supervisor: Prof Chris Ponting

Nov 2006 – June 2007, Institute of Evolutionary Biology, University of Edinburgh
•	Research Assistant
•	Supervisor: Prof Frank Jiggins

EDUCATION

Nov 2019 – Feb 2021
•	Fellowship of the Higher Education Academy (Edinburgh Teaching Award)

Oct 2007 – Aug 2011, Balliol College, University of Oxford
•	DPhil in Physiology, Anatomy and Genetics	
•	Supervisor: Prof Chris Ponting	
•	Thesis title: Evolution and Function of long noncoding RNAs in Drosophila

Sep 2005 – Aug 2006, University of Edinburgh
•	MSc in Quantitative Genetics and Genome Analysis	
•	Dissertation title: Evidence of selection in the genome of Staphylococcus aureus

Oct 2002 – June 2005, Trinity Hall, University of Cambridge
•	BA Hons Natural Sciences (Genetics)	Trinity Hall
•	First Class	University of Cambridge

TEACHING & EDUCATION PROVISION

•	Deliver lectures, practicals and tutorials across the Biomedical Informatics undergraduate degree programme at the Zhejiang University – University of Edinburgh (ZJE) Institute.  
•	Mark assessments (exams, essays, data analysis reports, presentations) across all courses where I deliver teaching at ZJE.  
•	Designed and presented workshop as part of the MRC Human Genetics Unit PhD student training programme describing the BEDTools bioinformatics tools.  
•	Supervised 16 Honours students (five UoE, 11 ZJE), frequently collaborating with experimental colleagues in projects focused on cancer and fungal genomics.  
•	Designed projects and supervised seven UoE taught MSc students in ‘Quantitative Genetics and Genome Analysis’ and ‘Bioinformatics’. My student in 2019 won the Douglas Falconer Award for gaining the top mark in her programme.
•	Nominated for EUSA ‘Supervisor of the Year’ in 2020, 2022. 

RESEARCH

•	Currently lead an independent, productive research programme of undergraduate and postgraduate students using functional, population and evolutionary genomics to investigate the noncoding regulation of medical phenotypes and disease risk. 
•	Regular reviewer for journals including BioEssays, BMC Genomics, Genome Biology & Evolution, Genome Research, Nucleic Acids Research, PLOS Genetics. 

•	Awarded £1,800 as the Principal Investigator of Moray Endowment Fund Grant to pilot functional genomics CAGE technology in previously collected blood samples from the VIKING Genes Study, 2021. 
•	Start-up funding of £14,000 associated with ZJE Lectureship to establish an independent research programme, 2019. 
•	Principal author of ‘molecular phenotypes’ section of successful Taylor group application for £2.4 million core funding at the MRC Human Genetics Unit, University of Edinburgh, 2018-22.  
•	Led successful application for a £500 Edinburgh University Small Grant to host the 4th Annual Scottish Biomedical Postdoctoral Researcher Conference, 2018.  

•	Primary supervisor for two UoE PhD students; co-supervisor for one UoE PhD student (primary supervisor Matt Brook); and co-supervisor for six ZJE PhD students (primary supervisors Mikael Bjorklund, Di Chen, Chan Kuan Yoow, Wanlu Liu, Stijn Van Der Veen, Chaochen Wang).  
•	Supervised one UoE MScR student in Infectious Diseases (Dissertation title: The Evolution of Covid-19 susceptibility and severity: did our evolutionary past drive genetics vulnerability to the pandemic?), 2021-22.
•	Supervised four ZJE MScR students in Integrative Biomedical Sciences, 2021-date. Three graduated students were each awarded a merit for high marks in their dissertations.  

KNOWLEDGE EXCHANGE AND IMPACT

•	Committee member for the UK Molecular Epidemiology Group since 2021. I helped organise a UK-wide conference in Edinburgh (2022) and ran an online flash-task competition for early-career researchers (2024).  
•	Provide mentorship (total number = 6) and interview practice (total number = 7) for CAMScot, an initiative to support Scottish state school pupils applying to Cambridge University.  
•	Regular volunteer for 4273pi, an organisation based at Edinburgh University which provides training in bioinformatics for high schools in the south of Scotland.  

ACADEMIC LEADERSHIP AND MANAGEMENT 

•	Course organiser for ‘Introduction to Biomedical Informatics 1’, a 20-credit course with 64 students run in semester 2 at ZJE.  
•	Member of the Course Assessment Group for courses at ZJE, including ‘Applied Data Science 2’ and ‘Genomics and Proteomics 2’.  
•	Member of the ZJE Special Circumstances Committee, 2022-24.  
•	Chair of PhD Thesis Committees at UoE (n = 1) and ZJE (n = 3).  

EXTERNAL RECOGNITION/ESTEEM

•	Trainee Award for Platform Presentation at the Human Genome Meeting, 2018. 
•	Full member of the UK Genetics Society since 2016.  

Oral Presentations
2024   *Quantitative Genetics and Genomics, Aarhus University	Online  
2023   *FANTOM6 Autumn Meeting	Milan, Italy  
2023   FANTOM6 Spring Meeting	Milan, Italy  
2023   *ZJE Annual Academic Conference	Online  
2018   Human Genome Meeting	Yokohama, Japan  
2017   Genome Informatics	Cold Spring Harbor, NY, USA  
2017   *FANTOM6 General Meeting	Yokohama, Japan  
2016   *Danish Bioinformatics Conference	Odense, Denmark  
2016   *11th Edinburgh Bioinformatics Meeting	Edinburgh, UK  
2015   International Mammalian Genome Conference	Yokohama, Japan  
2015   *FANTOM6 Strategy Meeting	Yokohama, Japan  
2014   Next Generation Sequencing Bioinformatics User Group	Aberdeen, UK  
2011   Oxford Fly Club	Oxford, UK  
* Invited speaker

Poster presentations  
2024  One Health Genomics Edinburgh Community Meeting	Edinburgh, UK  
2022  **American Society of Human Genetics Annual Meeting	Los Angeles, USA  
2021  Human Evolution – From Fossils to Ancient and Modern Genomes	Online  
2016  Scottish Biomedical Postdoctoral Research Conference	Edinburgh, UK  
2016   Edinburgh Genomics Conference	Edinburgh, UK  
2014   Chromatin: From Nucleosomes to Chromosomes	Cambridge, UK  
2010   Drosophila Research Conference	Washington, DC, USA  
2009   Biology of Genomes	Cold Spring Harbor, NY, USA  
** Reviewers’ Choice Abstract for top 10% abstracts  

PUBLICATIONS

* Teng J, Lin Q, Bai Z… **Young RS**… Fang L. The Farm Animal Genotype-Tissue Expression (FarmGTEx) Project (2025). Nature Genetics 57 (4): 786-96. doi: [10.1038/s41588-025-02121-5](https://doi.org/10.1038/s41588-025-02121-5).  
* Yu H, Wang Z, Ma J... **Young RS**... Chen D. The Establishment and Regulation of Human Germ Cell Lineage (2025). Stem Cell Research & Therapy 16 (1): 139. doi: [10.1186/s13287-025-04171-2](https://doi.org/10.1186/s13287-025-04171-2).  
*	Cornejo P, Petrova V, Zhang X, Young RS, Wong E. Emergence of Enhancers at late DNA Replicating Regions (2024). Nature Comms 15, 3451.  
*	Young RS, Talmane L, de Procé SM, Taylor MS. The Contribution of Evolutionarily Volatile Promoters to Molecular Phenotypes and Human Trait Variation (2022). Genome Biology 23 (1) 89.  
*	Ramilowski J, Yip CW, Agrawal S...Young RS...Carninci P. Functional Annotation of Human Long Non-Coding RNAs via Molecular Phenotyping (2020). Genome Res 39: 1060-1072. 
*	Alam T, Agrawal S, Severin J, Young RS...de Hoon MJL. Comparative transcriptomics of primary cells in vertebrates (2020). Genome Res 39: 951-961. 
*	Longman D, Jackson-Jones KA, Maslon MM, Murphy LC, Young RS, Stoddart JJ, Taylor MS, Papadopoulos DK, Cáceras JF. Identification of a localized nonsense-mediated decay pathway at the endoplasmic reticulum (2020). Genes Dev 34:1-14. 
*	Mordstein C, Savisaar R, Young RS, Bazile J, Talmane L, Luft J, Liss M, Taylor MS, Hurst L, Kudla G. Splicing buffers suboptimal codon usage in human cells (2020). Cell Systems 10: 351-362. 
*	Tan JY, Biasini A, Young RS, Marques AC. An unexpected contribution of lincRNA splicing to enhancer function (2020). Life Science Alliance 3 (4) e20200663.
*	Young RS, Kumar Y, Bickmore WA, Taylor MS. Bidirectional transcription initiation marks accessible chromatin and is not specific to enhancers (2017). Genome Biology 18 (1) 242.
*	Moss C, Dalla Rosa I, Hunt L… Young RS… Spinazzola A. Aberrant ribonucleotide incorporation and multiple deletions in mitochondrial DNA of the murine MPV17 disease model (2017). Nucleic Acids Research 45(22) 12808-12815.
*	Fernandez N*, Cordiner RA*, Young RS, Hug N, Macias S, Cáceras JF. Genetic variation and RNA structure regulate microRNA biogenesis (2017). Nature Communications 3 (8) 15114. 
*	Young RS. RNA meets DNA: The potential for gene expression to produce short RNA molecules capable of generating DNA mutation and driving genome evolution (Comment on DOI: 10.1002/bies.201700069) (2017). BioEssays 39 (10) 1700141.
*	Young RS. Lineage-specific genomics: Frequent Birth and Death in the Human Genome (2016). BioEssays 38 (7) 654-663.
*	Jubb AW*, Young RS*, Hume DA, Bickmore WA. Enhancer turnover is associated with a divergent transcriptional response to glucocorticoid in mouse and human macrophages (2016). Journal of Immunology 196(2): 813-822.
*	Young RS, Hayashizaki Y, Andersson R, Sandelin A, Kawaji H, Itoh M, Lassmann T, Carninci P, the FANTOM consortium, Bickmore WA, Forrest ARR, Taylor MS. The Frequent Evolutionary Birth and Death of Functional Promoters in Mouse and Human (2015). Genome Research 25(10): 1546-1557.
*	Forrest ARR, Kawaji H, Rehli M… Young RS (Contributing Author)… Hayashizaki Y. A Promoter-Level Mammalian Expression Atlas (2014). Nature 507: 462-470.
*	Mannion NM, Greenwood SM, Young RS… O’Connell MA. The RNA Editing Enzyme ADAR1 Controls Innate Immune Responses to RNA (2014): Cell Reports 9: 1-13.
*	Young RS and Ponting CP. Identification and Function of long noncoding RNAs (2013). Essays in Biochemistry 54: 113-126.
*	Young RS, Marques AC, Tibbit C, Bassett AR, Liu JL, Ponting CP. Identification and Characterisation of 1,119 lincRNA genes in Drosophila (2012). Genome Biology and Evolution 4(4): 427-442.
*	Bangham J, Knott SA, Kim KW, Young RS, Jiggins FM. Genetic variation affecting host-parasite interactions: major-effect quantitative trait loci affect the transmission of sigma virus in Drosophila melanogaster (2008). Molecular Ecology 17: 3800-3807.
*	Indicates co-first author

